server/config/validate-config-keys.js

```diff
@@ -46,7 +46,9 @@ const sharedConfig = parseConfig( '_shared.json' );
  * invalid. Such a missing value could and likely would
  * cause runtime errors in Calypso
  */
-environmentKeys.forEach( ( [ filename, keys ] ) => {
```

Comment:
Oh, was writing the response I didn't see your second message. I'm going to test #14631